front/app/containers/MainHeader/index.tsx

```diff
@@ -85,43 +90,24 @@ const MainHeader = () => {
   const containerRef = useRef<HTMLDivElement>(null);
   const locale = useLocale();
 
-  // url segments
-  const urlSegments = location.pathname.replace(/^\/+/g, '').split('/');
-  const firstUrlSegment = urlSegments[0];
-  const secondUrlSegment = urlSegments[1];
-  const lastUrlSegment = urlSegments[urlSegments.length - 1];
-
-  // Page checks
-  const isIdeaPage =
-    urlSegments.length === 3 &&
-    includes(locales, firstUrlSegment) &&
-    secondUrlSegment === 'ideas' &&
-    lastUrlSegment !== 'new';
-  const isInitiativePage =
-    urlSegments.length === 3 &&
-    includes(locales, firstUrlSegment) &&
-    secondUrlSegment === 'initiatives' &&
-    lastUrlSegment !== 'new';
-  const isAdminPage = isPage('admin', location.pathname);
-  const isProjectPage = !!(
-    urlSegments.length === 3 &&
-    urlSegments[0] === locale &&
-    urlSegments[1] === 'projects'
-  );
-
   const isSmallerThanTablet = useBreakpoint('tablet');
   const isDesktopUser = !isSmallerThanTablet;
 
+  // url segments
+  const urlSegments = location.pathname.replace(/^\/+/g, '').split('/');
+
   return (
     <Container
       id="e2e-navbar"
       className={`${
-        isAdminPage ? 'admin' : 'citizenPage'
+        isAdminPage() ? 'admin' : 'citizenPage'
```

Comment:
It seems like we don't use the admin className
```suggestion
        !isAdminPage() ? 'citizenPage' : ''
```